keyboards/wilba_tech/wt60_d/wt60_d.c

```diff
@@ -1,4 +1,20 @@
-#include "wt60_d.h"
+/* Copyright 2018 Jason Williams (Wilba)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
 
 void keyboard_pre_init_kb(void) {
     setPinOutput(F1);
```

Comment:
https://github.com/qmk/qmk_firmware/blob/cf45a0529671e33fdb63e3003d4340d7320b3e8e/keyboards/wilba_tech/wt60_d/wt60_d.c#L9-L14

This can be replaced by
```json
"incators": {
    "caps_lock": "F1"
},
```
and `wt60_d.c` file deleted too.